iacrcc/iacrcc.cls

```diff
@@ -154,24 +154,37 @@
 % doclicense added support for the macro \doclicenseLongTextForHyperref in version 3.1.0 (2022/04/27)
 \@ifpackagelater{doclicense}{2022/04/25}{\@doclicense@LongTextForHyperreftrue}{}
 \@ifpackageloaded{biblatex}{}{\bibliographystyle{alphaurl}}
+
 \AtEndPreamble{%
   \ifpdftex
     % TU encoding is only available with XeTeX and LuaTeX.
     % Defaulting to T1 encoding.
+    % fontenc: Standard package for selecting font encodings
     \RequirePackage[T1]{fontenc} 
   \fi
+  % hyperref: Extensive support for hypertext in LaTeX
+  \RequirePackage{hyperref}
   %% PDF title and author
   \if@anonymous
     \hypersetup{pdfauthor={hidden for submission}}
   \else
     \hypersetup{pdfauthor=\IACR@listofauthors}
   \fi
-  \hypersetup{pdftitle={\@plaintitle}}
   \if\@IACR@title@subtitle\empty
   \else
     \hypersetup{pdfsubtitle={\@IACR@title@subtitle}}
   \fi
-  % other PDF metadata
+  %% PDF keywords
+  \ifx\IACR@text@keywords\@empty\else
```

Comment:
Note one change here: previously we did not require the `\keywords` macro to appear in the preamble. We should update `iacrdoc.tex` to reflect this change.

Reply:
Argh, I prefer to fix the code and leave the usage of `\keywords` as-is. Let me push a fix.

Reply:
I'm not sure it's fixable. If \keywords appears after \begin{document} then it's too late to register the keywords in the XMP.

Reply:
Have a look at ff85a8c5d32f53a18d15a3ea425d8ff1273ec490, I confirmed this works for me.

Reply:
I verified that it doesn't matter whether \keywords is in the preamble or after `\begin{document}` using test14.